test/framework/src/main/java/org/elasticsearch/search/SearchResponseUtils.java

```diff
@@ -83,7 +84,7 @@ public enum SearchResponseUtils {
             SearchHit.METADATA_FIELDS,
             v -> new HashMap<String, DocumentField>()
         );
-        if (fieldName.equals(IgnoredFieldMapper.NAME)) {
+        if (IgnoredFieldMapper.NAME.equals(fieldName) || IgnoredSourceFieldMapper.NAME.equals(fieldName)) {
```

Comment:
Also `_ignored_source` is a multi-value field.

Reply:
I see that here we expand this logic from ignored to also ignored source. This change is also done in another place. So it is fair to say the ignored source field is the same kind of exception as ignored field?

Reply:
Yes because they can both return an "array" of values. This makes sense to me...in `_ignored` you have field names of fields which were ignored. In `_ignored_source` you have "tuples` of field names and values. So they are basically two arrays of the same size.